modules/tinymce/src/themes/silver/main/ts/ui/core/ComplexControls.ts

```diff
@@ -8,18 +8,18 @@
 import Editor from 'tinymce/core/api/Editor';
 
 import { UiFactoryBackstage } from '../../backstage/Backstage';
-import { alignSelectMenu } from './complex/AlignSelect';
-import { fontSelectMenu } from './complex/FontSelect';
-import { fontsizeSelectMenu } from './complex/FontsizeSelect';
-import { formatSelectMenu } from './complex/FormatSelect';
-import { styleSelectMenu } from './complex/StyleSelect';
+import { alignMenu } from './complex/AlignBespoke';
+import { blocksMenu } from './complex/BlocksBespoke';
+import { fontFamilyMenu } from './complex/FontFamilyBespoke';
+import { fontSizeMenu } from './complex/FontSizeBespoke';
+import { stylesMenu } from './complex/StylesBespoke';
```

Comment:
As you're changing these and they are a function... do you mind renaming all these bespoke menu functions to be something like `createAlignMenu`? That way it'd be consistent with the toolbar button function name as well.

Reply:
Done